pages/osx/asr.md

```diff
@@ -0,0 +1,20 @@
+# asr
+
+> Restore (copy) a disk image onto a volume.
+> The command name stands for Apple Software Restore.
+
+- Restore a disk image to a target volume:
+
+`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}}`
+
+- Erase the target volume before restoring:
+
+`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}} --erase`
+
+- Skip verification after restoring:
+
+`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}} -noverify`
```

Comment:
Is `-noverify` missing an extra dash `-`?

Reply:
you're right! good catch